client/lib/oauth2-clients.js

```diff
@@ -4,10 +4,18 @@
 
 import { includes } from 'lodash';
 
+export const isAkismetOAuth2Client = ( oauth2Client ) => {
+	return oauth2Client && oauth2Client.id === 973;
```

Comment:
[Optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) can simplify this line.
```suggestion
	return oauth2Client?.id === 973;
```